WooCommerce/Classes/POS/ViewModels/ItemSelectorViewModel.swift

```diff
@@ -23,26 +39,26 @@ final class ItemSelectorViewModel: ObservableObject {
 
     @MainActor
     func populatePointOfSaleItems() async {
-        isSyncingItems = true
```

Comment:
Shouldnt we have `state = .loading` here?
Or maybe set it in case `items` is empty?

Reply:
That makes sense, it's not needed right now because we only call this on init (which already has the initial state set to `.loading` and on `reload()`, which doesn't trigger the loading state but relies on the built-in loading indicator from `.refreshable()` 

That said, I agree that is better to add it and be explicit, since otherwise we're just relying that will be used as a side-effect of the existing implementation, which could change in the future. Updated here: https://github.com/woocommerce/woocommerce-ios/pull/13289/commits/06a499d15c8f4e63010ef0d1a32fb54545e3d95e